### PR TITLE
Complete milestone 1 of Project 3

### PR DESCRIPTION
**Pull request by YuvrajM108**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Culture</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Newsgeek</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Sports</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Health</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">The Debate</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Experts</h4></a>
+			<a class="category mid-cat" href="#"><h4 class="nav-text cat-text">Vantage</h4></a>
+			<a class="search"><img class="searchIcon" alt="Search Icon" src="https://icons-for-free.com/iconfiles/png/512/magnifying+glass+search+zoom+icon-1320196461375500720.png">
+			<h4 class="nav-text search">Search</h4>
+			</a>
```
Comment (Dipeshtwis) [inline index.html]: It should have **form/input** elements.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status: ♻️ Changes Required ♻️
## Hi @YuvrajM108 and Boaz, I'm Dipesh your reviewer for this PR.

Your clone is looking good and the project has good use of bootstrap classes. 👏 👏

## General Requirement

- [ ] I think there should be 2 contributors but right now I can see only 1. Both have to commit one by one after building each section or a specific feature. 
- [ ] You have to commit more often. Right now in one commit, you have written all the code. Please be aware of that.
- [ ] Make Readme and mention the project title and description. Also how to clone and run in a local environment. Think about the person who is coming from a non-technical background.

Now, coming to your project let me suggest you something so that we can make this project more awesome. I have mentioned some of the changes below.

## For Approval

## Header

- [ ] The Newsweek logo has a link.
- [ ] The login and subscribe should have a button type.
- [ ] In mobile, the Login button changes into an icon

## The main navigation (menu)

- [ ] Uses a nav tag
- [ ] It has a form with the search field. I have also mentioned it in the comments.
- [ ] In mobile and table, the menu hides and shows a hamburger button right to the subscribe button.
- [ ] There should be a border between two nav.

## Footer

- [ ] Newsweek Logo should have a link.
- [ ] just below the Newsweek logo It should be the logo (marked with orange).
- [ ] IN Tablet view, It is wrapping in itself (marked with blue). Please fix this.

Screenshot (1294)_LI.


Fix this and Request another code review.

## Reminder

**"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) now, you have only 2 code review left for this project.""**

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Aduda-Boaz & @YuvrajM108  :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- You've fixed most of the required changes by the last reviewer :+1: 
- All linter checks are passing successfully.

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] As mentioned by the last reviewer, your Readme is still empty. Please update it.
- [ ] On mobile view, the login button should be hidden completely. At the moment, it's still rendering with the icon.
- [ ] Try as much as possible to make sure that your clone looks similar to the original. For instance, the mobile view version of our footer section is quite different from what we have on the original page. Also, the hamburger menu is no rendering appropriately. Check the images below for details
Original page
original
original
Your clone
clone


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second-opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+					<a class="logo-link" href="#"><img class="logo" alt="Newsweek Logo" src="https://upload.wikimedia.org/wikipedia/commons/d/db/Newsweek_Logo.svg"></a>
+				</div>
+				<div class="col-5 d-flex justify-content-lg-center justify-content-around">
+					<div class="d-flex align-self-center">
+						<button class="nav-text login d-none d-md-inline" type="button">LOGIN</button>
+						<button class="login-icon d-md-none d-inline" type="button"><img class="login-icon" src="https://g.newsweek.com/www/images/user-w.svg" alt="login icon"></button>
+					</div>
+					<div class="d-flex align-self-center flex-nowrap">
+						<button class="nav-text subscribe" type="button"><span class="subButton bg-yellow">SUBSCRIBE &gt;</span></button>
+					</div>
```
Comment (Elbie-em) [inline index.html:27]: :heavy_exclamation_mark: The subscribe button should have a link

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+					<h4 class="nav-text date">Wed, Jan 20, 2021</h4>
+				</div>
+				<div class="col-lg-5 col-sm-3">
+					<a class="logo-link" href="#"><img class="logo" alt="Newsweek Logo" src="https://upload.wikimedia.org/wikipedia/commons/d/db/Newsweek_Logo.svg"></a>
+				</div>
+				<div class="col-5 d-flex justify-content-lg-center justify-content-around">
+					<div class="d-flex align-self-center">
+						<button class="nav-text login d-none d-md-inline" type="button">LOGIN</button>
+						<button class="login-icon d-md-none d-inline" type="button"><img class="login-icon" src="https://g.newsweek.com/www/images/user-w.svg" alt="login icon"></button>
+					</div>
```
Comment (Elbie-em) [inline index.html:24]: :heavy_exclamation_mark: The login should have a link

**Review by Elbie-em — APPROVED** [review]
## STATUS: APPROVED :heavy_check_mark:

Hello @YuvrajM108 and @Aduda-Boaz, good effort implementing changes requested by the TSE's and I am therefore approving this milestone.
However, I trust that you will make the changes outlined below and won't need to submit another code review. This will make your project stand out.

Kindly also check the comments and apply the changes requested.

## [OPTIONAL]

- [ ] It seems you have a major issue with spacing(padding and margin) in all resolutions. Use this 
[link](https://web.archive.org/web/20210120125445/https://www.newsweek.com/) as a reference to make sure your clone corresponds to the original.

> You can use the chrome inspection tool as a helper to guide you.

👏   All the best

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

File: index.html (hunk 2 above)
Comment (YuvrajM108) [reply index.html:27]: I get a linter error when I give the button a link.

**Pull request merged**